exist-core/src/main/java/org/exist/xquery/functions/fn/FunCurrentDateTime.java

```diff
@@ -94,7 +94,7 @@ public Sequence eval(Sequence contextSequence, Item contextItem) throws XPathExc
                 {context.getProfiler().message(this, Profiler.START_SEQUENCES,
                     "CONTEXT ITEM", contextItem.toSequence());}
         }
-        Sequence result = new DateTimeStampValue(context.getCalendar());
+        Sequence result = new DateTimeValue(context.getCalendar());
```

Comment:
Given my other comments, please verify if this change is needed.

Reply:
This has also now been undone.